packages/eslint-plugin/src/rules/no-magic-numbers.ts

```diff
@@ -55,6 +55,18 @@ export default util.createRule<Options, MessageIds>({
       return typeof node.value === 'number';
     }
 
+    /**
+     * Checks if the node grandparent is a Typescript enum declaration
+     * @param node the node to be validated.
+     * @returns true if the node grandparent is a Typescript enum declaration
+     * @private
+     */
+    function isGrandparentTSEnumDeclaration(node: TSESTree.Node): boolean {
```

Comment:
how come you're checking for the grandparent instead of just checking for the parent?
`TSEnumDeclaration` > `TSEnumMember` > `Literal`

Reply:
I assumed that would be the safest, I wasn't sure if there were other cases in which there would be a node type other than `TSEnumMember`, I'll change that.